chainer/links/normalization/group_normalization.py

```diff
@@ -49,10 +53,23 @@ def __init__(self, groups, size=None, eps=1e-5, initial_gamma=None,
         if initial_beta is None:
             initial_beta = 0
 
+        highprec_dtype = chainer.get_dtype(
+            None, map_mixed16=numpy.float32)
+
         with self.init_scope():
             self.groups = groups
-            self.gamma = variable.Parameter(initial_gamma)
-            self.beta = variable.Parameter(initial_beta)
+            if initial_gamma is None:
+                initial_gamma = 1
```

Comment:
This line is never executed since it's already checked for `None` above. You can remove the above 2 lines.